Application/Api/Education/Graduation/Evaluation/ApiEvaluation.php

```diff
@@ -50,18 +68,42 @@ public function exportApi($Method = '')
         $Dispatcher = new Dispatcher(__CLASS__);
         $Dispatcher->registerMethod('loadTestPlanning');
 
+        $Dispatcher->registerMethod('openGradeTextModal');
+        $Dispatcher->registerMethod('setGradeText');
+        $Dispatcher->registerMethod('changeGradeText');
+
         return $Dispatcher->callMethod($Method);
     }
 
     /**
      * @param string $Content
+     * @param string $Identifier
      *
      * @return BlockReceiver
      */
-    public static function receiverContent($Content = '')
+    public static function receiverContent($Content = '', $Identifier = '')
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Content" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)